Automattic/wp-calypso
# Stats: Adding skeleton loaders to stats module

Related to https://github.com/Automattic/red-team/issues/54

## Proposed Changes

* unifying TS types
* adding a skeleton loader for the new components

UI with the new skeleton loaders:
![SCR-20240704-meyd](https://github.com/Automattic/wp-calypso/assets/112354940/c9e728b5-3ede-45db-b3af-ebe1cd13fef3)




## Why are these changes being made?


* code maintenance and improving user experience

## Testing Instructions



* go to the live branch
* apply `stats/empty-module-traffic` feature flag
* navigate to several blogs and verify that the new components load properly

## Pre-merge Checklist
